db/seeds.rb

```diff
@@ -277,3 +277,13 @@
                   id: "6683c564-15e6-41af-ab44-7adf125f4c84",
                   jobseeker: jobseeker,
                   vacancy: physics_job)
+
+FactoryBot.create(:job_application, :status_shortlisted,
+                  id: "6683c564-15e6-41af-ab44-7adf125f4c85",
+                  jobseeker: jobseeker,
+                  vacancy: physics_job)
+
+FactoryBot.create(:job_application, :status_shortlisted,
```

Comment:
I'd err against creating seeds that don't make sense. A jobseeker can't apply for the same job twice, let alone be shortlisted twice for the same role

Reply:
sure was for my benefit only